lib/sensors/src/MoistureSensor.cpp

```diff
@@ -28,7 +26,7 @@ void MoistureSensor::Reset_Sensor() {
   i2c_.write(Sensor_I2C_Address, cmd, 3);  // set all registers on sensor to default values
```

Comment:
not pick: all private members should be prefixed with m_. dont append underscore imo. up to u tho

Reply:
The sensor address is a global variable defined above so it isn't exactly a private member. I think we could consider adding a variable for sensor addresses but it may be too vague to have a single variable represent many different types of addresses and having multiple different variables for all the possible types seems a bit clunky as well. @younesr1  what do you think. If it's not too clunky we could add a struct containing all the possible addresses that any sensor would need and just set the fields that we are using

Reply:
i meant that your i2c_ object is a bad name. the convention is to prefix private members, like i2c_ with m_ i think its been fixed since

Reply:
Wait this is fixed no?